concent_api/verifier/tasks.py

```
@@ -12,11 +24,49 @@
 @shared_task
 def blender_verification_order(
     subtask_id:             str,
-    source_package_path:    str,  # pylint: disable=unused-argument
-    result_package_path:    str,  # pylint: disable=unused-argument
+    source_package_path:    str,
+    result_package_path:    str,
     output_format:          Enum,  # pylint: disable=unused-argument
     scene_file:             str,  # pylint: disable=unused-argument
+    report_computed_task:   message.ReportComputedTask,
 ):
+    # Generate a FileTransferToken valid for a download of any file listed in the order.
+    file_transfer_tokens = {}
+
+    # !! Files MUST be added in following order - sources first, them results !!
+    for file_path in (source_package_path, result_package_path):
+        file_transfer_tokens[file_path] = create_file_transfer_token_for_concent(
```

Why are we still creating separate tokens? This should be a single token with both paths.

Is it just not updated yet after our discussions or is it a bug?

---

I thought what we discussed was only about Concent API view, but if it is applicable here too then it needs resolving of issue with lack of second 'size' and 'checksum'.

---

Well, there's no reason not to put multiple paths in a single token if you can.

Checksum for the source package is in TTC. The size is missing, yeah - but you're already generating the token here. What you putting in there?

After looking closer I now see that the function for generating the token is hard-coded to take RCT and you're giving it the same RCT when genrating a token for `source_package_path`. Which is a bug. If you don't have anything to give it, please don't use a wrong or made up value. If you have to use a placeholder, it should be something that fails loudly when someone tries to use it.

Yes, this issue should be blocked by the missing `size` field.

---

Task for adding `size` to `TaskToCompute` in golem-messages: https://github.com/golemfactory/golem-messages/issues/220